gistUploader.js

```diff
@@ -11,25 +11,17 @@ function buildPost(dir, files) {
     files: {},
   };
 
-  let filesProcessed = 0;
-
-  files.forEach((file, index, array) => {
-    fs.readFile(`${dir}/${file}`, 'utf8', (error, content) => {
-      if (error) throw error;
-      post.files[file] = {
-        content,
-      };
-
-      filesProcessed++;
-      if (filesProcessed === array.length) {
-        accessGitHub.postGist(post);
-      }
-    });
+  files.forEach((file) => {
```

Comment:
last refactor: when you see this pattern:
```
const accumulator = {}; // or let accumulator = [];
things.forEach(thing => { ...accumulator, thingName: thing })
```
you probably have a good case for `.reduce()`